openfisca_france/parameters/prelevements_sociaux/cotisations_securite_sociale_regime_general/red_m/assurance_maladie/sous_pss/employeur.yaml

```
@@ -0,0 +1,36 @@
+description: Taux employeur des cotisations de sécurité sociale (SS) des professions médicales à temps partiel (1961-1974), pour l'assurance maladie, sous le plafond de la sécurité sociale (PSS)
+unit: /1
+values:
+  1967-10-01:
+    value: 0.0605
+  1969-01-01:
+    value: 0.0665
+  1971-01-01:
+    value: 0.0732
+metadata:
+  ux_name: Taux employeur
+  last_review:
+    1971-01-01
+  reference:
+    1971-01-01: Arrêté du 30/12/1970
```

@Sasha-Laniece Quand la valeur n'est pas une URL, n'est-ce pas un `title` comme vu, par exemple, pour `openfisca_france/parameters/prelevements_sociaux/contributions_sociales/csg/activite/deductible/taux.yaml` ?
Ici, nous aurions ceci (ou sans href ? 🤔) : 
```suggestion
    1971-01-01: 
      - title: Arrêté du 30/12/1970
         href: null
```
Appliquer le principe de moindre surprise en affectant un unique type de valeur par clef ?